src/simulation/m_time_steppers.fpp

```diff
@@ -800,10 +800,6 @@ contains
             call s_pressure_relaxation_procedure(q_cons_ts(1)%vf)
         end if
 
-        call nvtxStartRange("RHS-ELASTIC")
-        if (hyperelasticity) call s_hyperelastic_rmt_stress_update(q_cons_ts(1)%vf, q_prim_vf)
```

Comment:
why was this removed?

Reply:
m_rhs.fypp calls this function already. 
This was originally there to provide you the stress state at the initial condition, but it is not necessary.
I added it back.